### ISSUE TITLE
Question about SyncBNHook

### ISSUE THREAD

**Issue by woxihuanjiangguo** [HuangJunJie2017/BEVDet#277]
Thanks for your consistent updates on this repo. I notice that in the update for occ task, there is a SyncBNHook implemented for models with STBase as their backbones. It is described that SyncBNHook can prevent loss divergency. Is there a connection between divergent loss and normal BatchNorm2D? I wonder if you can share with us your opinions on this conversion.

**Comment by HuangJunJie2017** [comment]
@woxihuanjiangguo Just empirical setting…… BN is used in both BEVEncoder and Head. The batch size in training STBase is too small to obtain stable statistics.

**Comment by woxihuanjiangguo** [comment]
i see. thank you anyway.
